### ISSUE TITLE
SDSS.query_photoobj fails for custom fields

### ISSUE THREAD

**Issue by digitaldingo** [astropy/astroquery#497]
There seem to be a problem retrieving custom fields in SDSS queries. Consider the following:

```python
from astroquery.sdss import SDSS
res = SDSS.query_photoobj(run=756, camcol=1, field=315,
                          fields = ["ra","dec","i",])
print(res)
```

`res` will only contain the "ra" and "dec" columns. Querying for the "i" column alone will throw an `IndexError`. 

This is not unique for "i" - it seems to me that any field (apart from the default ones) will cause this error.

**Comment by bsipocz** [comment]
I think this is a duplicate of #469.

**Comment by digitaldingo** [comment]
After reading that other issue more carefully I think you are right. The bug seems to be the same.

**Comment by bsipocz** [comment]
@keflavich - This was fixed with #508.
